ml_enabler/services/prediction_service.py

```diff
@@ -112,6 +112,9 @@ def get_aggregated_tiles(model_id: int, bbox: list, zoom: int):
         prediction_tiles = {}
         for prediction in predictions:
             # query all tiles within those quadkeys and aggregate
+            if int(prediction['tileZoom']) < int(zoom):
+                raise ValueError('Aggregate zoom level is lower than prediction zoom')
```

Comment:
Can we say "Aggregate zoom level is greater than prediction zoom" ? "Lower" seems a bit confusing to me - I know it's meant to mean that the zoom level is lower on the map.

Not sure what the clearest way to say this is, but if "lower" is the correct term here, we should change https://github.com/hotosm/ml-enabler-cli/blob/master/ml_enabler/aggregators/BaseAggregator.py#L10 to maintain consistency.

Reply:
Makes sense https://leafletjs.com/examples/zoom-levels/ -- I always confuse this and use the way one looks at the earth from space. Lower you go the more granular. But let's stick to the definition on leaflet and others. Thanks @batpad